tests/baselines/reference/callSignatureAssignabilityInInheritance6.errors.txt

```diff
@@ -45,8 +45,6 @@ tests/cases/conformance/types/typeRelationships/assignmentCompatibility/callSign
           Types of property 'a' are incompatible.
             Type 'T' is not assignable to type 'T'. Two different types with this name exist, but they are unrelated.
               'T' could be instantiated with an arbitrary type which could be unrelated to 'T'.
-                Type 'Base' is not assignable to type 'T'.
-                  'T' could be instantiated with an arbitrary type which could be unrelated to 'Base'.

```

Comment:
One thing wich I'm not a 100% certain about based on the original issue, is whether we want to continue elaborating if the target and source type are the same. Like in this case where both are `T`. What do you think?